packages/web-components/src/components/toggle-tip/toggletip.ts

```diff
@@ -98,9 +117,10 @@ class CDSToggletip extends HostListenerMixin(FocusMixin(LitElement)) {
     return html`
       <button
         aria-controls="${this.id}"
+        aria-label="Show information"
```

Comment:
Suggest creating a `label` property (or something similar), which defaults to "Show information" but can be modified by users for localization.